packages/resource-deployment/scripts/upload-files.sh

```diff
@@ -23,7 +27,7 @@ uploadFileBatch() {
 exitWithUsageInfo() {
     echo \
         "
-Usage: $0 -s <storage account name> -d <path to drop folder (optional)>
+Usage: $0 -s <storage account name> -d <path to drop folder. default value: $dropFolder>
```

Comment:
>default value: $dropFolder [](start = 61, length = 26)

Can you replace with similar:
"Will use the '$dropFolder' folder when run from the current location"